monGARS/api/web_api.py

```
@@ -61,6 +62,44 @@ async def conversation_history(
+
+
+class ChatResponse(BaseModel):
+    response: str
+    confidence: float
+    processing_time: float
+
+
+@app.post("/api/v1/conversation/chat", response_model=ChatResponse)
+async def chat(
+    chat: ChatRequest,
+    current_user: dict = Depends(get_current_user),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)